### ISSUE TITLE
./otherpkgs: line 931: /usr/bin/logger: Argument list too long

### ISSUE THREAD

**Issue by kcgthb** [xcat2/xcat-core#5644]
The `otherpkgs` postscript sometimes generates an error when too many arguments are passed to `logger`:
https://github.com/xcat2/xcat-core/blob/4164dd2dc9501fa0b17f6f812bbec7b6118e707a/xCAT/postscripts/otherpkgs#L931

The generated error looks like this in `/var/log/xcat/xcat.log`
```
./otherpkgs: line 931: /usr/bin/logger: Argument list too long
Wed Sep 19 11:06:11 PDT 2018 [info]: xcat.mypostscript: postscript otherpkgs return with 0
```

**Comment by robin2008** [comment]
The log message size exceed the limitation as it contains the full yum command output.
Perhaps we can split the log into two parts for yum/zypper/apt installation result:
- Using ERROR/INFO to log the result of packages installation, including command line 
- Using DEBUG to log detail results,  we can leverage PIPE to split the large message into single line messages

For the detail results,  it is better only be logged when DEBUG mode, verbose mode or the result is error.

**Comment by robin2008** [comment]
@bybai   You can consider the above enhancement in syslog consolidation.

**Comment by robin2008** [comment]
@kcgthb  the new fix is in #5743

**Comment by bybai** [comment]
It is fixed in #5743, I will close this issue, if needed, re-open it later.
